### DISCUSSION TITLE
KeY is on Github.

### DISCUSSION THREAD

**Discussion by wadoon** [KeYProject/key#3000] [Announcements]
After long years in our own [Gitlab instance](https://git.key-project.org), we decided to open the  development of KeY and moved the sources to Github. Thus, you can (more) easily communicate with us about issues or enhancement in our theorem prover. The [homepage](https://key-project.org) and Gitlab instance remains, also the mailing list `support@key-project.org`. 

KeY is a platform for the deductive verification of Java programs with specification written in the Java Modeling Language, which allows you to prove the correctness of source code. 

In the next weeks, we will configure our organization and repository to make them a little more comfortable.

Regards, 

Alexander

**Comment by Neelu35** [comment]
- @mlooz []()
